platform/ios/src/MGLLocationManager.m

```diff
@@ -22,6 +22,7 @@ @implementation MGLLocationManager
 - (instancetype)init {
     self = [super init];
     if (self) {
+        [MGLConfig.sharedConfig configurationFromKey:[[NSUserDefaults standardUserDefaults] objectForKey:MGLMapboxMetricsProfile]];
```

Comment:
The call to `-objectForKey:` will usually return nil since it is unlikely a developer would have set a value for the `MGLMapboxMetricsProfile` key.

Reply:
I think we should make this call to set up `MGLConfig` from a more centralized location. Maybe when `MGLMapboxEvents` is initialized? https://github.com/mapbox/mapbox-gl-native/blob/rclee-radius-config-agua/platform/ios/src/MGLMapboxEvents.m#L172

This is ok for now but future configuration will not necessarily be limited to only MGLLocationManager behavior.